packages/compass-home/src/components/home.tsx

```diff
@@ -219,7 +237,7 @@ function Home({ appName }: { appName: string }): React.ReactElement | null {
 
   useEffect(() => {
     // Setup listeners.
-    appRegistry.on('instance-refreshed', onInstanceRefreshed);
+    appRegistry.on('instance-created', onInstanceCreated);
```

Comment:
is this just giving a better name or the semantic is actually different? Like this event is meant to be happen only once?

Reply:
This even is different from the previous one. The `instance-refreshed` is called every time we "refresh" the instance which currently includes 1) instance created for the first time when user connects and initial data is fetched 2) user does something that triggers an instance refresh: pressing a refresh button, adding/removing documents, running an aggregation with an $out step, and some other things I don't remember right away.

This new `instance-created` event is emitted only when user connects and we create the instance, at this point we have not even started the initial fetch/refresh call. This gives other stores in the app the opportunity to subscribe to the partial state tree changes (e.g., only in database or collection they are interested in) instead of just flushing their state every time something in the app caused a state refresh